guava/src/com/google/common/primitives/UnsignedBytes.java

```diff
@@ -103,7 +107,7 @@ public static byte checkedCast(@IntRange(from = 0, to = 255) long value) {
    * @return {@code (byte) 255} if {@code value >= 255}, {@code (byte) 0} if {@code value <= 0}, and
    *     {@code value} cast to {@code byte} otherwise
    */
-  public static byte saturatedCast(long value) {
+  public static @Unsigned byte saturatedCast(@UnknownSignedness long value) {
```

Comment:
The documentation says what happens if `value` is less than zero.  That means that `value` is treated as signed.

Reply:
@mernst Yes you are right. I have resolved this!.